test/unittests/state_mpt_hash_test.cpp

```diff
@@ -58,3 +62,191 @@ TEST(state_mpt_hash, deleted_storage)
+    tx.data =
+        "04a7e62e00000000000000000000000000000000000000000000000000000000000000c0000000000000000000"
+        "000000000000000000000000000000000000000000024000000000000000000000000000000000000000000000"
+        "0000000000000000028000000000000000000000000000000000000000000000000000000000000002c0000000"
+        "000000000000000000000000000000000000000000000000000000030000000000000000000000000000000000"
+        "000000000000000000000000000003400000000000000000000000000000000000000000000000000000000000"
+        "000001ba90df364951119f0e935b90ed342b9e686985fb7805f532c5432c2a46ba1233be5ed196ab7d467c8cc0"
+        "73686342699c000000000000000000000000000000000000000000000000000000000000000000000000000000"
+        "000000000063ecd7e7000000000000000000000000000000000000000000000000000000000000000200000000"
+        "0000000000000000000000000000000000000000000000000000000a0000000000000000000000000000000000"
+        "000000000000000000000000000001000000000000000000000000aafb72183a85a66ec7eec6a9d3374f3a06d8"
+        "a25100000000000000000000000000000000000000000000000000000000000000010000000000000000000000"
+        "000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"
+        "00000000000000040000000000000000000000000000000000000000000000054c7bff9ff28e80000000000000"
+        "000000000000000000000000000000000000000000000000000001964617c9cbc649c28b9710bbe61cc10e0000"
+        "000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"
+        "01ba90df364951119f0e935b90ed342b9e686985fb7805f532c5432c2a46ba1233000000000000000000000000"
+        "000000000000000000000000000000000000000155f6604df131609d8058c7f0ad8bbdf96f4bb6b5cc00c96aad"
+        "da6f61455681990000000000000000000000000000000000000000000000000000000000000001000000000000"
+        "000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000"
+        "000000000000000000000000010000000000000000000000000000000000000000000000054c7bff9ff28e8000"
+        ""_hex;
+
+    tx.gas_limit = 387780;
+    tx.max_gas_price = 1500000014;
+    tx.max_priority_gas_price = 1500000000;
+    tx.sender = 0x3a091a68661d40dafc2a532f8ba89ad2c0b4f184_address;
+    tx.to = 0xacd9a09eb3123602937cb30ff717e746c57a5132_address;
+    tx.value = 0;
+    AccessList access_list = {};
+    tx.nonce = 10246;
+    tx.r = 0xdf2ff0c61a24ece7b4c24d9a1a7061881043fd8285ea0be8ea55b42c8a119225_u256;
+    tx.s = 0x644cd7390b5f274ee947121837da3deab1638c0c7d9f5aa4ebe9f9a3149f192d_u256;
+    tx.v = 1;
+    tx.chain_id = 11155111;
+
+    std::array transactions{tx};
+    auto tx_root = mpt_hash(transactions);
+
+    EXPECT_EQ(tx_root, 0x6ce50bfaaebabe884433c144fa4d8a4c1087e443587a9788b30381636dedbeb2_bytes32);
+}
+
+TEST(state_mpt_hash, legacy_and_eip1559_receipt_three_logs_no_logs)
+{
+    // https://sepolia.etherscan.io/tx/0x1e68d9dbbf933399a6dfe5686ba0b51e04a4da81ab17aa5ff84334fdf2d4a3a7
+
+    //    {
```

Comment:
Where do you have JSON from? If you have it already, maybe just JSON load it?

Reply:
From JSON RCP calling. I try to load.

Reply:
Done